targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064B0S2_4343W/policy/policy_multi_CM0_CM4_tfm.json

```
@@ -0,0 +1,219 @@
+                "encrypt_key": "../keys/image-aes-128.key",
+                "encrypt_key_id": 1,
+                "encrypt_peer": "../keys/dev_pub_key.pem",
+                "upgrade_auth": [
+                    6
+                ],
+                "upgrade_keys": [
+                    { "kid": 6, "key": "../keys/TFM_S_KEY.json" }
+                ],
+                "resources": [
+                    {
+                        "type": "BOOT",
+                        "address": 268435456,
+                        "size": 327680
+                    },
+                    {
+                        "type": "UPGRADE",
+                        "address": 269942784,
+                        "size": 327680
+                    }
+                ]
+            },
+            {
+                "boot_auth": [
+                    8
+                ],
+                "boot_keys": [
+                    { "kid": 8, "key": "../keys/TFM_NS_KEY.json" }
+                ],
+                "id": 16,
+                "monotonic": 8,
+                "smif_id": 1,
+                "multi_image" : 2,
+                "upgrade": true,
+                "version": "0.1",
+                "rollback_counter": 0,
+                "encrypt": false,
+                "encrypt_key": "../keys/image-aes-128.key",
+                "encrypt_key_id": 1,
+                "encrypt_peer": "../keys/dev_pub_key.pem",
+                "upgrade_auth": [
+                    8
+                ],
+                "upgrade_keys": [
+                    { "kid": 8, "key": "../keys/TFM_NS_KEY.json" }
+                ],
+                "resources": [
+                    {
+                        "type": "BOOT",
+                        "address": 268763136,
+                        "size": 1146880
+                    },
+                    {
+                        "type": "UPGRADE",
+                        "address": 402653184,
+                        "size": 1146880
+                    }
+                ]
+            }
+        ],
+        "reprogram": [
+            {
+                "start": 270336000,
+                "size": 65536
+            }
+        ],
+        "reprovision": {
+            "boot_loader": true,
+            "keys_and_policies": true
+        }
+    },
+    "cy_bootloader":
+    {
+        "mode": "debug"
+    },
+    "provisioning":
+    {
+        "packet_dir": "../packets",
+        "chain_of_trust": []
+    },
+    "pre_build": {
+        "oem_public_key": "../keys/oem_state.json",
+        "oem_private_key": "../keys/oem_state.json",
+        "hsm_public_key": "../keys/hsm_state.json",
+        "hsm_private_key": "../keys/hsm_state.json",
+        "provision_group_private_key": false,
+        "group_private_key": "../keys/grp_priv_key.json",
+        "provision_device_private_key": false,
+        "device_private_key": "../keys/dev_priv_key.json",
+        "cy_auth": "../packets/cy_auth_2m_s0_sample.jwt"
```

is it should be cy_auth_2m_**b0**_sample.jwt?
```suggestion
        "cy_auth": "../packets/cy_auth_2m_b0_sample.jwt"
```